utils.py

```diff
@@ -26,15 +26,16 @@ def __init__(
         self.tag = tag
         self.target = target
 
-    def get_pkg_build_cmd(self) -> List[str]:
+    def get_pkg_build_cmd(self, tag_or_branch: str) -> List[str]:
         return [
             "bash",
             "-c",
             " && ".join(
                 [
-                    "pip install './dvc[all]'",
-                    "pip install -r dvc/scripts/build-requirements.txt",
-                    f"python dvc/scripts/build.py {self.pkg}",
+                    f"git clone --branch '{tag_or_branch}' https://github.com/iterative/dvc /dvc",
```

Comment:
One problem with this approach is that we are no longer able to debug locally by building our own local dvc copy

Reply:
Maybe we can chmod/chown the clone instead?

Reply:
Found a workaround: `git config --global --add safe.directory '*'`